### DISCUSSION TITLE
Displayfehler

### DISCUSSION THREAD

**Discussion by anpalama1** [tbnobody/OpenDTU#995] [General]
bei dem 1,3" Display SH1106   leuchtet dauerhaft die rechte Randseite. Der gleiche Fehler tritt bei anderen Displays auf.   Nur bei dem kleineren 0,9" Display kommt der Fehler nicht vor. Liegt der Fehler dann am Display oder kann er softwareseitig behoben werden?

  
![20230603_091439](https://github.com/tbnobody/OpenDTU/assets/120574638/1c1a52af-3a28-4ae1-b4a5-b97aa0b09640)

**Comment by teletapi** [comment]
Dann hast du in der Pinmapping den falschen Displaytyp stehen da muss Typ3 stehen beim sh1106 1.3"
0,96 Display SSD1306 ist Typ 2

**Reply by Parateam** [reply]
Bestätigt, war bei mir genauso. Ist jetzt OK
Ich dachte schon das Display ist defekt
